File: sdkwork-ai-sdk-csharp/Models/Message.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class Message
    {
        public string? Role { get; set; }
        public object? Content { get; set; }
        public string? Refusal { get; set; }
        public List<object>? Annotations { get; set; }
        public string? ReasoningContent { get; set; }
        public List<ToolCall>? ToolCalls { get; set; }
    }
}

File: sdkwork-ai-sdk-go/types/message.go
package types

// 消息
type Message struct {
	Role string `json:"role"`
	Content interface{} `json:"content"`
	Refusal string `json:"refusal"`
	Annotations []interface{} `json:"annotations"`
	ReasoningContent string `json:"reasoning_content"`
	ToolCalls []ToolCall `json:"tool_calls"`
}
